README.md

```diff
@@ -69,6 +69,8 @@ which chromedriver
 xattr -d com.apple.quarantine /path/to/chromedriver
 ```
 
+If this has no effect, you may need to try another path, for example: `/usr/local/bin/chromedriver`
```

Comment:
😕  `/path/to` is just a placeholder. Not a big Homebrew fan in general, but I know I'm in a total minority amongst Mac users and we assume it elsewhere in our READMEs, so let's just replace the placeholder with `/usr/local/bin`?

Reply:
I realise that /path/to is a placeholder, but when I replaced it with the output of `which chromedriver`, there was no effect.

Reply:
```bash
$ which chromedriver
/Users/daivdmears/.local/bin/chromedriver
```

Reply:
Ooh interesting – how did you end up with two Chromedrivers! And even more interestingly, how is this one before the Homebrew one in the `PATH`, yet the Homebrew one is the one used by your tests 🧐

Reply:
Maybe by being silly. I'll undo this part of the changes to the README anyway.